doc/source/ray-core/compiled-graph/overlap.rst

```diff
@@ -1,5 +1,7 @@
-Overlap communication and computation
-======================================
+.. _compiled-graph-overlap:
+
+Experimental: Overlapping communication and computation
+=======================================================
 
 Compiled Graph currently provides experimental support for GPU communication and computation overlap. When you turn this feature on, it automatically overlaps the GPU communication with computation operations, thereby hiding the communication overhead and improving performance.
```

Comment:
curious, why is this an experimental feature // not turned on by default? is there a behavior difference, or just not stable yet?

Reply:
It's not stable yet.